src/pages/o/[orgId]/events/[eventId].tsx

```diff
@@ -69,80 +82,80 @@ const OrgEventPage : PageWithLayout<OrgEventPageProps> = (props) => {
     const org = orgQuery.data as ZetkinOrganization;
 
     return (
-        <Flex direction="column">
-            <Heading data-test="event-title" level={ 1 }>
-                { event.title ? event.title : event.activity.title }
-            </Heading>
-            <Flex>
-                <Text marginEnd="size-50">
-                    <Msg id="pages.orgEvent.details.org"/>:
-                </Text>
+        <>
+            <Header marginBottom="size-300">
+                <Image
+                    alt="Cover image"
+                    height="size-2000"
+                    objectFit="cover"
+                    src="/cover.jpg"
+                    width="100%"
+                />
+                <Heading data-test="event-title" level={ 1 } marginBottom="size-50">
+                    { event.title ? event.title : event.activity.title }
+                </Heading>
                 <Link>
                     <NextLink href={ `/o/${orgId}` }>
                         <a>{ org.title }</a>
                     </NextLink>
                 </Link>
-            </Flex>
-            <Flex>
-                <Text marginEnd="size-50">
-                    <Msg id="pages.orgEvent.details.campaign"/>:
-                </Text>
+            </Header>
+            <Flex marginBottom="size-100">
+                <Flag aria-label="S Flag" marginEnd="size-100" size="S"/>
```

Comment:
Area the aria labels required? If they are, they should probably be internationalized. Otherwise I think we should remove them in this case, where the icon is really just decoration.